orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/domain/MinionIpAddressesResponse.java

```diff
@@ -26,6 +26,15 @@
                 .collect(Collectors.toList());
     }
 
+    @JsonIgnore
+    public List<String> getUnreachableNodes() {
+        return result.stream()
+                .flatMap(result -> result.entrySet().stream())
+                .filter(entry -> "false".equals(entry.getValue().asText()))
```

Comment:
is it possible that `entry.getValue()` is null? if yes, then maybe it is worth to filter them out

Reply:
Hopefully not, the same logic is used in the above method, it should have failed already.

Reply:
Ok, it makes sense